2-mandatory/1-create-functions.js

```diff
@@ -24,18 +31,45 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(arr) {
+  const newArr = [];
+  const finalArr = [];
+
+  for (let word of arr) {
+    newArr.push(word.trim());
+  }
+
+  for (let word of newArr) {
+    let strArr = word.split("");
+
+    for (let l of strArr) {
+      if (l == "/") {
+        delete strArr[strArr.indexOf(l)];
+      }
+    }
+    let join = strArr.join("");
+    finalArr.push(join.toLowerCase());
+  }
+  return finalArr;
 }
-
 /*
 Write a function that:
 - Takes an array and an index as input.
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, index) {
+  const newArr = arr.slice(0, arr.length);
+  const finalArr = [];
+  for (let num of newArr) {
+    if (newArr.indexOf(num) == index) {
```

Comment:
well done! I liked using the indexOf( ) here!

Reply:
Thank you @DelniaAlipour!